Intermediate Lighting Launch

helencarnate opened the pull request:
Two down... two to go. Blood looked at this locally for an extra set of eyes.

megalon commented on wiki/mapping/intermediate-lighting.md:
```diff
@@ -0,0 +1,117 @@
+:::
+**On/Off** and **Flash/Off** strobes are considered "true" strobes, alternating bright light and darkness in rapid succession. Strobing should be used cautiously as overuse can make it very difficult to concentrate on your map or even harm players with neurological conditions.
+
+As discussed in the [Flash/On](/mapping/intermediate-lighting.md#flash-on) section, "on" lights are at 100% brightness while "Flash" lights begin at 110% and quickly intensify to 125%. Flash/Offs produce a more intense strobe than On/Offs.
```
I think this Flash/On link is broken

helencarnate commented:
Both issues updated. Fixed Flash/On link and added a link from the Lighting Practices section on mapping home. Thanks, as always, for your second (in this case third! set of eyes.
